.gitignore

```diff
@@ -32,3 +32,6 @@
 
 /node_modules
 .vscode/settings.json
+
```

Comment:
Your project works pefectly!
[OPTIONAL] Kindly add MIT license to your project.